java/jbmc-regression/CharSequenceToString/Main.java

```diff
@@ -11,6 +11,8 @@
 public class Main {
   public static void main(String[] args) {
     String arg = Verifier.nondetString();
+    if (arg == null)
```

Comment:
:no_entry_sign: This shoudn't be necessary. `nondetString` is defined in a way never to return null.